src/rendering/dashboard.cpp

```diff
@@ -49,6 +49,15 @@ namespace {
         "in x and y-direction on screen.",
         openspace::properties::Property::Visibility::User
     };
+
+    constexpr openspace::properties::Property::PropertyInfo RefreshRateInfo = {
+        "RefreshRate",
+        "Refresh Rate (in ms)",
+        "If this value is bigger than 0, the value represents the number of milliseconds "
+        "between refreshs of the dashboard items. If the value is 0 the dashbaord is "
+        "refreshed at the same rate as the main rendering.",
```

Comment:
```suggestion
        "The number of milliseconds between refreshes of the dashboard items. If the value is 0 
        "the dashboard is refreshed at the same rate as the main rendering.",
```
?

Reply:
Note that if you want to keep the old phrasing, there are also some typos
```suggestion
        "If this value is larger than 0, the value represents the number of milliseconds "
        "between refreshes of the dashboard items. If the value is 0 the dashboard is "
        "refreshed at the same rate as the main rendering.",
```